backend/sftp/options_test.go

```diff
@@ -338,7 +338,7 @@ func (o *optionsSuite) TestGetAuthMethods() {
 				KeyFilePath:   o.keyFiles.SSHPrivateKey,
 				KeyPassphrase: o.keyFiles.passphrase,
 				Password:      "somepassword",
-				KeyExchanges:  "diffie-hellman-group-exchange-sha256",
+				KeyExchanges:  []string{"diffie-hellman-group-exchange-sha256"},
```

Comment:
Did we consider adding a test case with multiple exchanges to ensure that functions as expected beyond the type change?

Reply:
added in this commit! https://github.com/C2FO/vfs/pull/102/commits/254f491d7d0fcde2ed2b1db17fc50193fa0a9cd9